src/js/modifier.js

```diff
@@ -0,0 +1,11 @@
+import { setState } from './state.js'
+import { render } from './index.js'
+
+const reset = () => {
+  setState('input', '')
+  setState('inputArr', ['0'])
+
+  render()
+}
+
+export { reset }
```

Comment:
```suggestion
export const reset = () => {
  setState('input', '')
  setState('inputArr', ['0'])

  render()
}
```
이렇게 표현할 수 있답니다!

그리고 파일 명은 modifier 보다 reset.js 처럼 함수와 일치시켜주면 더 좋지 않을까요?